tests/Nox.Types.Tests/Enum/NoxTypeTests.cs

```diff
@@ -0,0 +1,30 @@
+﻿using FluentAssertions;
+using FluentAssertions.Execution;
+
+namespace Nox.Types.Tests.Enum
+{
+    public class NoxTypeTests
+    {
+        [Fact]
+        public void NoxType_Value_Is_Nuid()
+        {
+            const string noxTypeFormatNuid = "Nox.Types.{0}";
+
+            using (new AssertionScope())
+            {
+                foreach (var noxType in System.Enum.GetValues<NoxType>())
+                {
+                    var id = Utilities.Identifier.Nuid.ToInt32(string.Format(noxTypeFormatNuid, noxType.ToString()));
+
+
+                    ((uint)noxType).Should().Be(ToUint32(id));
+                    //Console.WriteLine($"{noxType.ToString()} = {ToUint32(id)},");
+                }
+            }
+        }
+        private static uint ToUint32(int nuid)
+        {
+            return (uint)nuid + int.MaxValue;
```

Comment:
This is a bug:

The formula for converting from 'int' -> 'uint':

return (uint)(nuid +int.MaxValue+1)